src/python/pants/backend/python/packaging/pyoxidizer/rules.py

```diff
@@ -121,7 +159,16 @@ async def package_pyoxidizer_binary(
             output_directories=["build"],
         ),
     )
+    process_with_caching = dataclasses.replace(
+        pex_process,
+        argv=(bash.path, runner_script.path, *pex_process.argv),
+        append_only_caches={
+            **pex_process.append_only_caches,
+            "pyoxidizer": os.path.join(".cache", "pyoxidizer"),
```

Comment:
It would probably be good for the script to share this value instead of ~duplicating it.